src/lib/Legislator.svelte

```diff
@@ -130,8 +130,11 @@
   }
 
```

Comment:
What if we add a media query to toggle the old behavior on larger screens?
```suggestion
@media screen and (min-width: 1199px) {
  .committees-grid-item {
    grid-column: auto;
  }

  .committees {
    list-style-type: none;
    padding-left: 0;
  }
}
```
![committee_list_02](https://github.com/code4sac/sacramento-campaign-finance/assets/6665595/05e19f66-486f-423b-8690-cf4a28d45571)
![committee_list_01](https://github.com/code4sac/sacramento-campaign-finance/assets/6665595/b85204a0-2aff-4bf0-a42f-8455038e16a3)

Reply:
The xl desktop screenshot still contains the bullet list style. The suggested code keeps `list-style-type: none;` which I forgot when taking the screenshot. I think the list looks better without bullets on the wide layout.

Reply:
I'm open to it, feel free to open as a PR? I think the bullets are kind of nice because all of the letters are capitalized so it's hard to tell where one committee name starts and another ends sometimes.